manifest.json

```diff
@@ -1017,7 +1017,7 @@
       "react-native-exif@0.4.0",
       "react-native-view-shot@2.6.0",
       "react-native-camera@1.13.1",
-      "react-native-permissions@1.1.1",
+      "react-native-permissions@2.0.10",
```

Comment:
Quick question here: What's the reason `react-native-permissions` is only being updated for platform version `0.37` and later? Looking at the rest of this file, it was first introduced in platform version `0.15`.

Reply:
I tested the container generation for versions 0.37 through the latest :) So only updated these.

Yes, this can go to the earlier versions too. I just checked the directives up to version 13 of the platform & this configuration should work.